gammapy/scripts/image_pipe.py

```diff
@@ -2,56 +2,180 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 import logging
+import numpy as np
+from astropy.coordinates import Angle
+from astropy.units import Quantity
+from astropy.io import fits
 from ..utils.scripts import get_parser
+from ..background import fill_acceptance_image
+from ..image import SkyMap, SkyMapCollection, disk_correlate
+from ..stats import significance
 
 __all__ = ['ImageAnalysis']
 
 log = logging.getLogger(__name__)
 
 
-def image_pipe_main(args=None):
-    parser = get_parser(ImageAnalysis)
-    parser.add_argument('config_file', type=str,
-                        help='Config file in YAML format')
-    # TODO: add option to dump the default config file
-    args = parser.parse_args(args)
-    analysis = ImageAnalysis.from_yaml(args.config_file)
-    analysis.run()
-
-
 class ImageAnalysis(object):
     """Gammapy 2D image based analysis.
+
+    Parameters
+    ----------
+    empty_image : `~gammapy.image.SkyMap`
```

Comment:
@cdeil 
Indeed, it was better to add this reference!!
I'm not sure of what I did in the init function about that:
```
else:
 +            self.data_store = data_store
 +            self.energy_band = energy_band
 +            self.offset_band = offset_band
 +
 +            self.empty_image = empty_image
 +            self.header = self.empty_image.to_image_hdu().header
```
It's just that we don't need these members if the user has already a counts and bkg maps but this is not well coded I think?